app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,10 +6,28 @@ def index
   end
 
   def create
-    # binding.pry
   end
 
   def show
     @error = current_site.issues.where('issues.id = ?', params[:id]).first
   end
+
+   def update
+    @error = Issue.find(params[:id])
+    @error.update_attributes(error_params)
+  end
+
+  def notify_subscribers
+    @error = Issue.find(params[:id])
+    @message = params[:message]
+    Website.find(@error.website_id).subscribers.each do |member|
```

Comment:
schimba in `@error.subscribers` nu e nevoie de inca un query ca sa iei subscriberi.